### ISSUE TITLE
bug in guide

### ISSUE THREAD

**Issue by BenfenYU** [spring-guides/gs-messaging-stomp-websocket#68]
Hi, I find there is a bug in guide "https://spring.io/guides/gs/messaging-stomp-websocket/#initial",
I choose to download the repository, and then follow doc to "Jump ahead to Create a Resource Representation Class.", however the repository does not include the "pom.xml" with modifications on dependencies. So maybe the guide should be modified to "Jump ahead to Starting with Spring Initializr."
![image](https://user-images.githubusercontent.com/33017944/100594008-16df9700-3334-11eb-8175-2dc30a7694ce.png)

**Comment by Buzzardo** [comment]
I think you're right. Since it's easy to fix, I'm going to add the `good first issue` and `help wanted` labels and see if someone in the community would like to tackle it.

**Comment by EvandroMaddes** [comment]
I did't find this issue in the current repo. I think it's a problem that only affects the description present on [spring.io](https://spring.io/guides/gs/messaging-stomp-websocket)

**Comment by EvandroMaddes** [comment]
yes, you're right but in the [readme ](https://github.com/spring-guides/gs-messaging-stomp-websocket/blob/main/README.adoc) file this issue is not present, so where i can solve it?

**Comment by Buzzardo** [comment]
Oh, I see now. The link is in an included block of content. We don't dare change it there, because it would break other guides. The deep fix here is to make sure all the guides are consistent with their `scratch` and `initial` identifiers and then change the link in that included file. I'll have to look into that, and I'll close this one. Thanks for trying to address an issue and for making me look deeply enough to find the other issue.
